goreleaser/docker.yml

```diff
@@ -190,81 +190,81 @@ dockers:
           - LICENSE
           - licenses
 
-    # arm64 builds follow
```

Comment:
TODO: revert this change before merging, it's just to make it build on my machine